enforcer-maven-plugin: copyright check is failing on binary files

## Description

copyright check is failing on the Helidon workspace with the following exception:

```
Caused by: java.nio.charset.MalformedInputException: Input length = 1
    at java.nio.charset.CoderResult.throwException (CoderResult.java:279)
    at sun.nio.cs.StreamDecoder.implRead (StreamDecoder.java:405)
    at sun.nio.cs.StreamDecoder.lockedRead (StreamDecoder.java:217)
    at sun.nio.cs.StreamDecoder.read (StreamDecoder.java:177)
    at java.io.BufferedReader.fill (BufferedReader.java:160)
    at java.io.BufferedReader.implReadLine (BufferedReader.java:370)
    at java.io.BufferedReader.readLine (BufferedReader.java:353)
    at java.io.BufferedReader.readLine (BufferedReader.java:436)
    at java.nio.file.FileChannelLinesSpliterator.readLine (FileChannelLinesSpliterator.java:187)
    at java.nio.file.FileChannelLinesSpliterator.tryAdvance (FileChannelLinesSpliterator.java:117)
    at java.util.stream.ReferencePipeline.forEachWithCancel (ReferencePipeline.java:147)
    at java.util.stream.AbstractPipeline.copyIntoWithCancel (AbstractPipeline.java:588)
    at java.util.stream.AbstractPipeline.copyInto (AbstractPipeline.java:574)
    at java.util.stream.AbstractPipeline.wrapAndCopyInto (AbstractPipeline.java:560)
    at java.util.stream.MatchOps$MatchOp.evaluateSequential (MatchOps.java:230)
    at java.util.stream.MatchOps$MatchOp.evaluateSequential (MatchOps.java:196)
    at java.util.stream.AbstractPipeline.evaluate (AbstractPipeline.java:265)
    at java.util.stream.ReferencePipeline.anyMatch (ReferencePipeline.java:672)
    at io.helidon.build.common.FileUtils.containsLine (FileUtils.java:1152)
    at io.helidon.build.maven.enforcer.FileFinder.isValid (FileFinder.java:180)
    at io.helidon.build.maven.enforcer.FileFinder.lambda$findFiles$0 (FileFinder.java:97)
```

The file being processed is `docs/src/main/asciidoc/images/guides/12_tracing_top.png` which is a binary file.

https://github.com/helidon-io/helidon-build-tools/blob/f210a893fa4d4b1a465f7715a4a8fa6860ca70eb/maven-plugins/enforcer-maven-plugin/src/main/java/io/helidon/build/maven/enforcer/FileFinder.java#L180

https://github.com/helidon-io/helidon-build-tools/blob/f210a893fa4d4b1a465f7715a4a8fa6860ca70eb/common/common/src/main/java/io/helidon/build/common/FileUtils.java#L1151

This regression was introduced in https://github.com/helidon-io/helidon-build-tools/pull/1114

---

`[submodule` is a syntax used in the `.gitmodules` file,  I'm not sure I understand the requirement to read the lines of every file being validated.

The default behavior of `git log` is to ignore submodules (see `git log --help`  for `--ignore-submodule`).

---

We should remove the offending code.
We can consider passing `--ignore-submodule=all` to discard explicit configuration in the `.gitmodules` file.

--- a/maven-plugins/enforcer-maven-plugin/src/main/java/io/helidon/build/maven/enforcer/FileFinder.java
+++ b/maven-plugins/enforcer-maven-plugin/src/main/java/io/helidon/build/maven/enforcer/FileFinder.java
@@ -33,7 +33,6 @@ import java.util.stream.Collectors;
 
 import io.helidon.build.common.logging.Log;
 
-import static io.helidon.build.common.FileUtils.containsLine;
 import static io.helidon.build.maven.enforcer.GitIgnore.create;
 
 /**
@@ -174,11 +173,6 @@ public class FileFinder {
 
         if (Files.isSymbolicLink(file.path())) {
             Log.debug("File " + file.relativePath() + " is a symbolic link, ignoring.");
-            return false;
-        }
-
-        if (containsLine(file.path(), line -> line.startsWith("[submodule"))) {
-            Log.debug("File " + file.relativePath() + " is a submodule, ignoring.");
             return false;
         }
 

